image_classification/mobilenet_nchw.js

```diff
@@ -17,25 +17,34 @@ export class MobileNetV2Nchw {
       labelUrl: './labels/labels1000.txt',
       inputDimensions: [1, 3, 224, 224],
     };
+    this.constantsAarry_ = [];
```

Comment:
Could you explain why we need to this unused array? I mean add an annotation.

Reply:
@huningxin, could you please take a look on this workaround specific for node env? This would make user quite confused.

Reply:
> @huningxin, could you please take a look on this workaround specific for node env? This would make user quite confused.

@junwei As I know, you are trying to optimize webnn-native implementation codes to resolve this problem, right? Then I will update accordingly. Thanks!

Reply:
We should fix the issue in webnn-native and remove this workaround here in the sample.